main.c

```diff
@@ -323,6 +325,55 @@ const pattern_t unlabeled = {
     .get_color = get_color_stripes
 };
 
+const pattern_t aromantic = {
+    .name = "aromantic",
+    .ansii_pattern = {
+      .codes_count = 10,
+      .ansii_codes = {
+        34, 34,
+        120, 120,
+        255, 255,
+        247, 247,
+        233, 233
+      }
+    },
+    .color_pattern = {
+      .stripes_count = 5,
+      .stripes_colors = {
+          0x3da542, // Green
+          0xa8d379, // Light green
+          0xffffff, // White
+          0xa9a9a9, // Grey
+          0x000000  // Black
```

Comment:
Make the comments the same as in the other patterns, with `/* #value - Color */`